2020-06-17 20:38:13,016 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2020-06-17 20:38:13,018 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Python38\lib\site-packages\instabot\bot\bot.py
2020-06-17 20:38:13,018 - instabot version: 0.117.0 (api_login) - DEBUG - Loading cookies
2020-06-17 20:38:13,019 - instabot version: 0.117.0 (api_login) - DEBUG - Loading uuids
2020-06-17 20:38:13,019 - instabot version: 0.117.0 (api_login) - INFO - Recovery from D:\python_files\InstaBot/config/viivek_uvv_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
- user-agent=Instagram 117.0.0.28.123 Android (28/9.0; 420dpi; 1080x1920; OnePlus; ONEPLUS A3003; OnePlus3; qcom; en_US; 180322800)
- phone_id=798076f4-f73d-4a32-a16f-c9c77aba0e74
- uuid=92240e82-69b6-43a5-b3b1-6cb53e26b937
- client_session_id=14d49a25-75d9-4a0a-9977-28f5fd2a8de1
- device_id=android-0134f5818f0fb323
2020-06-17 20:38:13,029 - instabot version: 0.117.0 (api_login) - INFO - LOGIN FLOW! Just logged-in: False
2020-06-17 20:38:14,579 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: feed/timeline/ returned response: <Response [200]>
2020-06-17 20:38:15,855 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: feed/reels_tray/ returned response: <Response [200]>
2020-06-17 20:38:15,865 - instabot version: 0.117.0 (api) - INFO - Logged-in successfully as 'viivek_uvv'!
2020-06-17 20:38:15,867 - instabot version: 0.117.0 (bot_checkpoint) - DEBUG - Loading Checkpoint file from: D:\python_files\InstaBot/config/viivek_uvv.checkpoint
2020-06-17 20:38:16,555 - instabot version: 0.117.0 (api) - DEBUG - GET to endpoint: users/viswa5427/usernameinfo/ returned response: <Response [200]>
2020-06-17 20:38:18,114 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: direct_v2/threads/broadcast/upload_photo/ returned response: <Response [403]>
2020-06-17 20:38:18,114 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"action": "item_ack", "status_code": "403", "payload": {"client_context": "76cdaa1f-799d-4659-bd08-d3382112512a", "message": "Client upgrade needed."}, "status": "fail"}'
2020-06-17 20:38:18,115 - instabot version: 0.117.0 (api) - ERROR - Request returns 403 error!
2020-06-17 20:38:18,117 - instabot version: 0.117.0 (bot_direct) - INFO - Message to ['10140409752'] wasn't sent
2020-06-17 20:38:18,119 - instabot version: 0.117.0 (bot_checkpoint) - DEBUG - Saving Checkpoint file to: D:\python_files\InstaBot/config/viivek_uvv.checkpoint
2020-06-17 20:38:18,122 - instabot version: 0.117.0 (bot) - INFO - Total requests: 36
